src/lib/Server/Output/ValueObjectVisitor/Policy.php

```diff
@@ -21,20 +21,18 @@ class Policy extends ValueObjectVisitor
     /**
      * Visit struct returned by controllers.
      *
-     * @param \Ibexa\Contracts\Rest\Output\Visitor $visitor
-     * @param \Ibexa\Contracts\Rest\Output\Generator $generator
-     * @param Policy|\Ibexa\Contracts\Core\Repository\Values\User\PolicyDraft $data
+     * @param \Ibexa\Contracts\Core\Repository\Values\User\PolicyDraft|\Ibexa\Contracts\Core\Repository\Values\User\Policy} $data
```

Comment:
Trailing `}` after the type hint. Not sure if we need to do union here, since PolicyDraft extends Policy and the polymorphic distinction is handled via L46-49.
```suggestion
     * @param \Ibexa\Contracts\Core\Repository\Values\User\Policy $data
```